stubs/inertia/resources/js/Jetstream/AuthenticationCardLogo.vue

```diff
@@ -0,0 +1,8 @@
+<template>
+    <a href="/">
+        <svg class="w-16 h-16" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Couldn't we just reuse the `ApplicationMark` component here instead of duplicating?

Reply:
Yeah, I looked at that, but it's different, and because I'm trying to replicate the other views to keep branding consistent I copied this one instead.